<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditWindow</class>
 <widget class="QDialog" name="EditWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>156</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="idTxt">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Id</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="titleTxt">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Title</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="categoryTxt">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Category</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ratingTxt">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Rating</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="priceTxt">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Price</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="quantityTxt">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>80</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Quantity</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="stockCheck">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>93</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>In stock</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
